distribution/lib/Standard/Table/0.0.0-dev/src/Column.enso

```diff
@@ -2341,6 +2341,28 @@ type Column
 
        Arguments:
        - range: The selection of rows from the table to return.
+
+       Returns:
+       A new Column containing the selected rows.
+
+       ! Error Conditions
+       - If the range is out of bounds, an `Index_Out_Of_Bounds` error is raised.
+
+       > Example
+         Select the first row from the "Name" Column.
+
+             table = Table.new [["Name", ["Alice", "Bob", "Charlie"]]]
+             column = table.get "Name"
+             ## The take returns "Alice"
+             first_row = column.take (First 1)
+
+       > Example
+         Select the last row from the "Name" Column.
+
+             table = Table.new [["Name", ["Alice", "Bob", "Charlie"]]]
+             column = table.get "Name"
+             ## The take returns "Charlie"
+             last_row = column.take (Last 1)
```

Comment:
I'm wondering if it would be simpler to have a sample that does not rely on table:
```suggestion
             column = Column.from_vector "Name" ["Alice", "Bob", "Charlie"]
             ## The take returns "Charlie"
             last_row = column.take (Last 1)
```

On the other hand, I know that `Table` is one of the 'base' types that our users will be familiar with, so I guess a more table-centric approach is still OK.

Reply:
Another thing - it feels like it would be good to try formatting the examples in a way that will be compatible with the automated example testing that we are planning to prepare.
So maybe something like this could work?
```suggestion
             table = Table.new [["Name", ["Alice", "Bob", "Charlie"]]]
             column = table.get "Name"
             last_row = column.take (Last 1)
             last_row.to_vector
             >>> ["Charlie"]
```
Or something similar to indicate the expected output of the last expression?

Reply:
We could also aim for a standardized text comment to make it more human-readable:
```suggestion
             table = Table.new [["Name", ["Alice", "Bob", "Charlie"]]]
             column = table.get "Name"
             last_row = column.take (Last 1)
             ## This will return: ["Charlie"]
             last_row.to_vector
```

Reply:
Happy with this will return and that's exactly the point of the comment. I'll be building auto testing into the doc gen